CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 * Updated gem dependency to support Rails 5.0.x.
 
+### enhancements
+  * Support to Rails 5.0.x Stable. [@herminiotorres](https://github.com/herminiotorres)
+  * Removed deprecated methods `default_input_size`. [@herminiotorres](https://github.com/herminiotorres)
+  * Update `country_select` to version `2.5.2`. [@herminiotorres](https://github.com/herminiotorres)
```

Comment:
`country_select` isn't a runtime dependency here, so the value for reporting this back to developers that come to our CHANGELOG for updates on Simple Form isn't much helpful - the gem already works with `country_select`, we are just updating our test suite for it, right?